lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -249,43 +249,48 @@ class Color {
   @override
   int get hashCode => value.hashCode;
 
-  /// Converts color to a css compatible attribute value.
-  // webOnly
-  String toCssString() {
-    if ((0xff000000 & value) == 0xff000000) {
-      return toCssStringRgbOnly();
-    } else {
-      final double alpha = ((value >> 24) & 0xFF) / 255.0;
-      final StringBuffer sb = StringBuffer();
-      sb.write('rgba(');
-      sb.write(((value >> 16) & 0xFF).toString());
-      sb.write(',');
-      sb.write(((value >> 8) & 0xFF).toString());
-      sb.write(',');
-      sb.write((value & 0xFF).toString());
-      sb.write(',');
-      sb.write(alpha.toString());
-      sb.write(')');
-      return sb.toString();
-    }
-  }
-
-  /// Returns the CSS value of this color without the alpha component.
-  ///
-  /// This is useful when painting shadows as on the Web shadow opacity combines
-  /// with the paint opacity.
-  // webOnly
-  String toCssStringRgbOnly() {
-    final String paddedValue = '00000${value.toRadixString(16)}';
-    return '#${paddedValue.substring(paddedValue.length - 6)}';
-  }
-
   @override
   String toString() {
     return 'Color(0x${value.toRadixString(16).padLeft(8, '0')})';
   }
 }
 
+/// Converts color to a css compatible attribute value.
```

Comment:
it looks like all usages are actually in `engine`, so we can probably move this to engine utils, remove the `ui.` prefix, and remove the `// webOnly` comment

Reply:
Done. PTAL